### PR TITLE
Usage of deprecated commands would crash due to nil AppData

### PR DESCRIPTION
**Pull request by andydotxyz**
This is an issue also for usage of the not-deprecated `Getter`

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: cmd/fyne/internal/commands/release.go [hunk 1]
```diff
@@ -149,6 +148,13 @@ type Releaser struct {
 	password     string
 }
 
+// NewReleaser returns a command that can handle the packaging a GUI apps for release from local Fyne source code.
+func NewReleaser() *Releaser {
+	r := &Releaser{}
+	r.appData = &appData{}
+	return r
```
Comment (Jacalz) [inline cmd/fyne/internal/commands/release.go:155]: Should we not do it like this to be consistent with the others?
```suggestion
	return &Releaser{appData: &appData{}}
```

**Review by Jacalz — CHANGES_REQUESTED** [review]
Thanks. Just one minor change for consistency.

File: cmd/fyne/internal/commands/release.go (hunk 1 above)
Comment (andydotxyz) [reply cmd/fyne/internal/commands/release.go:155]: No, that won't work - the appData is not a field of Releaser but inherited from a parent.

Comment (Jacalz) [reply cmd/fyne/internal/commands/release.go:155]: Ah, I see. Alright then. That is honestly one of the few things with Go that I strongly dislike

**Review by Jacalz — APPROVED** [review]

**Pull request merged**